src/OpenLoco/Windows/PromptBrowseWindow.cpp

```diff
@@ -683,20 +688,30 @@ namespace OpenLoco::Ui::Windows::PromptBrowse
         _files.clear();
         if (_currentDirectory.empty())
         {
-            _files = platform::getDrives();
+            // Get all drives then remove all unusable drives such as CD-ROM drives with no cd in them
+            for (const auto& drive : platform::getDrives())
+            {
+                // Unready drives will generate an error code and are classed as not directories
+                std::error_code ec;
+                if (fs::is_directory(drive, ec))
```

Comment:
The error code is unused. Does it make a difference if we retrieve it?

Perhaps we should handle this in `platform::getDrives` instead?

Reply:
The error code is just so we don't throw an exception on unready drives. It's basically just faster. If we moved it into the platform::getDrives then it will be very slow any time you want to get the drives. I'm not sure if that's good idea or not.

Reply:
Hmm i could use winapi calls to do it though in platform! Okay will redesign this

Reply:
I tried out a few winapi ideas and none were very good so all ive done is move the logic into platform.